### ISSUE TITLE
No perturbation on 3rd layer

### ISSUE THREAD

**Issue by patrillicit** [a-lucic/cf-gnnexplainer#6]
src/cf_explanation/gcn_perturb.py

self.gc1 = GraphConvolutionPerturb(nfeat, nhid)
self.gc2 = GraphConvolutionPerturb(nhid, nhid)
**self.gc3 = GraphConvolution(nhid, nout)**
self.lin = nn.Linear(nhid + nhid + nout, nclass)
self.dropout = dropout

in the GCNSyntheticPerturb class when the model layers are built the first two layers are the perturbed layers, while the 3rd one keeps being a simple convolution layer. 

Why is that?
